packages/blade/src/components/Amount/amountTokens.ts

```diff
@@ -0,0 +1,85 @@
+import type { FontSize } from './../../tokens/global/typography';
+import type { AmountProps } from './Amount';
+
+const affixFontSizes: Record<NonNullable<AmountProps['size']>, keyof FontSize> = {
+  'body-small': 75,
+  'body-small-bold': 75,
+  'body-medium': 75,
+  'body-medium-bold': 75,
+  'heading-small': 75,
+  'heading-small-bold': 75,
+  'heading-large': 100,
+  'heading-large-bold': 100,
+  'title-small': 300,
+  'title-medium': 400,
+} as const;
+
+const amountTextSizes: Record<NonNullable<AmountProps['size']>, keyof FontSize> = {
```

Comment:
Lets keep the naming consistent. We're calling it `affixFontSizes` above, so lets call it FontSize here also
```suggestion
const amountFontSizes: Record<NonNullable<AmountProps['size']>, keyof FontSize> = {
```